client/js/libs/handlebars/parse.js

```diff
@@ -1,126 +1,318 @@
 "use strict";
 
-Handlebars.registerHelper(
-	"parse", function(text) {
-		text = Handlebars.Utils.escapeExpression(text);
-		text = colors(text);
-		text = channels(text);
-		text = uri(text);
-		return text;
-	}
-);
-
-function uri(text) {
-	return window.URI.withinString(text, function(url) {
```

Comment:
i need start & end position of the url to correctly merge it with the styling information.
i'll look at it.